docs/config-file-package-configuration.yml.md

```diff
@@ -0,0 +1,101 @@
+# The package configuration file (Experimental)
+
+A package configuration sets up path excludes and license finding curations specific to a package (dependency) and 
+provenance. It is thus a similar concept to the `.ort.yml` file which allows to do similar configuration for projects as
+opposed to packages.
+
+## Specifying the target package and provenance
```

Comment:
Add `# Package Configuration File Basics` to align with documentation lay-out in other doc file